examples/BUCK

```diff
@@ -3,9 +3,6 @@ cxx_binary(
   srcs = [
     'main.cpp',
   ],
-  compiler_flags = [
```

Comment:
> ...are you sure about this? ctti requires C++14.

This is handled via the `.buckconfig` file, which allows you to specify top-level configuration such as the language standard and compiler used. It is better to put this in the `.buckconfig` than the target itself because mixing-and-matching standards can lead to ABI incompatibilities. By placing that information in the `.buckconfig` it can be controlled for all targets at once. 

I can confirm that the `BUCK` script works on Clang + macOS and Linux + GCC. 

I don't have Windows to hand, but given how clean the project structure is Buck will simply be passing the translation-units to MSVC, so it should be fine. Is there an Appveyor file in the repo? I can't see it.